lib/ansible/plugins/inventory/gcp_compute.py

```diff
@@ -120,16 +120,10 @@
 import json
 
 from ansible.errors import AnsibleError, AnsibleParserError
-from ansible.module_utils._text import to_native
+from ansible.module_utils._text import to_text
 from ansible.module_utils.gcp_utils import GcpSession, navigate_hash, GcpRequestException
 from ansible.plugins.inventory import BaseInventoryPlugin, Constructable, Cacheable
 
-try:
-    import google.auth
-    import requests
-except ImportError:
-    raise AnsibleError('The gcp dynamic inventory plugin requires the requests and google-auth libraries')
-
```

Comment:
Removing this changes the behavior. Even though there weren't used directly, it was used to determine if the correct libs were installed.

You could instead add `from ansible.module_utils.gcp_utils import HAS_GOOGLE_LIBRARIES`, but I believe raising the error is still useful.

Reply:
should still not be raised on import, but on parse